src/style.css

```
@@ -1 +1,8 @@
 /* styles go here */
+label:not(:first-child) {
+  display: block;
+  margin-top: 20px;
+}
+.form-button {
```

Add blank lines between rules
```suggestion
}

.form-button {
```

---

code corrected